roles/ovirt-repositories/tasks/rpm.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Backup repositories
+  include: backup-repos.yml
```

Comment:
Should this be optional, It no where said it's doing this, so please also document it

Reply:
ok